### ISSUE TITLE
New Admin UI Toggle/Switch elements defaults to true position for non defined properties/options

### ISSUE THREAD

**Issue by DAHAG-ArisNourbakhsh** [keycloak/keycloak-ui#3425]
### Describe the bug

In the new Admin UI the switch toggle defaults to the true/"On" position for options that are undefined and have no value.

### Version

19.0.1

### Expected behavior

Keycloak internally treats non defined properties as false and the UI should reflect that by showing toggles as "Off" for that. (The old Admin UI correctly handles the display of how it should work.)

For example in this `Client scope` `User client scope` Mapper the `id.token.claim` is not included and thus the toggle is set to "Off"

![client-roles-old-UI](https://user-images.githubusercontent.com/104843708/187458193-237ffd5e-508c-4b42-a7be-ed0b7102a264.JPG)

### Actual behavior

Switching to the new theme the toggle will be on the on position for the same mapper:

![client-roles-newUi](https://user-images.githubusercontent.com/104843708/187460081-409064e3-6abe-41dc-9d2a-8af50248d90b.JPG)

### How to Reproduce?

Seed a new blank Keycloak  and take a look at the auto generated `Master Realm` -> Client Scopes -> `roles` -> `realm roles` Mapper for example. API returns no value for `id.token.claim` so it should be "Off"

**Comment by ssilvert** [comment]
Note: Whoever works on this, make sure we have a universal solution.

**Comment by edewit** [comment]
So I've looked at this and actually this seems to be an issue in the old console, rather then an issue in the new. These screens are created based on the server info. Defined for this mapper there is this:

```json
{
   "id":"oidc-usermodel-client-role-mapper",
   "name":"User Client Role",
   "category":"Token mapper",
   "helpText":"Map a user client role to a token claim.",
   "priority":40,
   "properties":[
      {
         "name":"usermodel.clientRoleMapping.clientId",
         "label":"usermodel.clientRoleMapping.clientId.label",
         "helpText":"usermodel.clientRoleMapping.clientId.tooltip",
         "type":"ClientList",
         "secret":false,
         "readOnly":false
      },
      {
         "name":"usermodel.clientRoleMapping.rolePrefix",
         "label":"usermodel.clientRoleMapping.rolePrefix.label",
         "helpText":"usermodel.clientRoleMapping.rolePrefix.tooltip",
         "type":"String",
         "secret":false,
         "readOnly":false
      },
      {
         "name":"multivalued",
         "label":"multivalued.label",
         "helpText":"multivalued.tooltip",
         "type":"boolean",
         "defaultValue":"true",
         "secret":false,
         "readOnly":false
      },
      {
         "name":"claim.name",
         "label":"tokenClaimName.label",
         "helpText":"usermodel.clientRoleMapping.tokenClaimName.tooltip",
         "type":"String",
         "secret":false,
         "readOnly":false
      },
      {
         "name":"jsonType.label",
         "label":"jsonType.label",
         "helpText":"jsonType.tooltip",
         "type":"List",
         "options":[
            "String",
            "long",
            "int",
            "boolean",
            "JSON"
         ],
         "secret":false,
         "readOnly":false
      },
      {
         "name":"id.token.claim",
         "label":"includeInIdToken.label",
         "helpText":"includeInIdToken.tooltip",
         "type":"boolean",
         "defaultValue":"true",
         "secret":false,
         "readOnly":false
      },
      {
         "name":"access.token.claim",
         "label":"includeInAccessToken.label",
         "helpText":"includeInAccessToken.tooltip",
         "type":"boolean",
         "defaultValue":"true",
         "secret":false,
         "readOnly":false
      },
      {
         "name":"userinfo.token.claim",
         "label":"includeInUserInfo.label",
         "helpText":"includeInUserInfo.tooltip",
         "type":"boolean",
         "defaultValue":"true",
         "secret":false,
         "readOnly":false
      }
   ]
}
```
As you can see there are default values for these switches that the old console seems to ignore, but the new console takes the default value if it's set.

**Comment by DAHAG-ArisNourbakhsh** [comment]
I'd say that, that is an issue with the new console. 
(I included the response from the network tab in the screenshots)

`id.token.claim` is not included in the reponse. Keycloak, in it's backend logic will treat the absence of `id.token.claim` as "false" but the new UI "ignores" that and applies a default value to the toggle control.

**Comment by edewit** [comment]
fixed by #3819
